DscResources/MSFT_xMySqlGrant/MSFT_xMySqlGrant.schema.mof

```diff
@@ -2,12 +2,12 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlGrant")] 
 class MSFT_xMySqlGrant : OMI_BaseResource
 { 
-  [Key, Description("Name of MySQL user.")] String UserName;
-  [Key, Description("MySql database name to grant permissions.")] String DatabaseName;
-  [Required, Description("MySql connection credential used for the root."), EmbeddedInstance("MSFT_Credential")] String ConnectionCredential;
-  [Write, Description("MySql user permission type."), ValueMap{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}, Values{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}] String PermissionType;
-  [Write, Description("Ensure given grant to mySql database present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-
+    [Key, Description("Name of MySQL user.")] String UserName;
+    [Key, Description("MySql database name to grant permissions.")] String DatabaseName;
+    [Required, Description("MySql connection credential used for the root."), EmbeddedInstance("MSFT_Credential")] String RootCredential;
+    [Key, Description("MySql user permission type."), ValueMap{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}, Values{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}] String PermissionType;
```

Comment:
If we specified a default value for the version we would then have to constantly update this resource as the new resource came out, we would also then force people to upgrade to the latest version if they used the default value. Forcing the user to specify the version they would want strikes me as a better idea as upgrades or non-compliance to a version should be driven from the user.